lib/core/algorithm/decorator/persistence_decorator.dart

```diff
@@ -35,8 +35,10 @@ class PersistenceDecorator extends ForceDecorator {
   void compute(Vertex v, Graph graph) {
     super.compute(v, graph);
     Vertex? cachedVertex = _positionMap[v];
-    if (cachedVertex == null || cachedVertex.position != v.position) {
+    if (cachedVertex == null || cachedVertex.position.x != v.position.x || cachedVertex.position.y != v.position.y ) {
```

Comment:
Can you run "dart format ." and resubmit it again? If possible.
There is a formatting issue here.